robotbase/src/main/java/com/team2357/frc2022/subsystems/SubsystemFactory.java

```diff
@@ -44,16 +45,16 @@ public FalconTrajectoryDriveSubsystem CreateFalconTrajectoryDriveSubsystem() {
         return subsystem;
     }
 
-    public IntakeSubsystem CreateIntakeSubsystem() {
+    public IntakeSubsystem CreateIntakeSubsystem(ArduinoSensorState state) {
```

Comment:
Variable name of `state` is not descriptive. Suggest something like `intakeCargoSensor` instead.